CMakeLists.txt

```diff
@@ -296,18 +296,7 @@ include (ParsecCompilerFlags)
 include (cmake_modules/CheckAtomicIntrinsic.cmake)
 if(CMAKE_SYSTEM_NAME MATCHES "Darwin")
   set( PARSEC_OSX 1 CACHE INTERNAL "Compile on MAC OS X")
```

Comment:
I don't think we need PARSEC_OSX anymore, as far as I see all remaining uses can rely on the default \_\_APPLE__ instead.